.github/workflows/verify_version.yaml

```diff
@@ -0,0 +1,20 @@
+name: Verify Release
+
+on:
+  pull_request:
+    branches: [main, "v[0-9]+.[0-9]+.[0-9]+"]
```

Comment:
isn't the branch is just v[0-9]+.[0-9] ?

Reply:
Not necessarily, I'd try to catch any semver, but the comment is legit because this is not catching v1.0 :-) 
Will change it